config/default-spinnaker-local.yml

```diff
@@ -45,6 +45,22 @@ providers:
     #   name: my-cf-username
     #   password: my-cf-password
 
+  kubernetes:
+    # If you want to deploy some services to Kubernetes
+    # set enabled and provide primary credentials for deploying.
+    # Enabling google is independent of other providers.
+    enabled: ${SPINNAKER_GOOGLE_ENABLED}
+    primaryCredentials:
+      # These credentials require a username and password for the Kubernetes
+      # master endpoint. You can supply an optional list of namespaces that
+      # can be deployed to by Spinnaker (defaults to ["default"]). They will be
+      # created if they don't exist.
+      name: my-kubernetes-account
```

Comment:
Should this account name match the account name used in these sample commands (https://github.com/spinnaker/clouddriver/blob/master/clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/ops/DeployKubernetesAtomicOperation.groovy#L47-L49)?